### ISSUE TITLE
[BUG] [REGRESSION] Brightmaps don't load on DOOM and DOOM2 IWADS. They do still work on FREEDOOM IWADS.  

### ISSUE THREAD

**Issue by blackhand1001** [ZDoom/gzdoom#2760]
### GZDoom version

4.12.2

### Which game are you running with GZDoom?

Doom 2

### What Operating System are you using?

Windows 11

### Please describe your specific OS version

Windows 11 Pro 24H2

### Relevant hardware info

14900KF - RTX 4090

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Brightmaps do not load when playing any of the ID Software IWADS. Freedoom based iwads load them correctly. 

It occurs on 4.12.2 and newer. It works correctly on 4.11.3 so the regression occured between those builds. 

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Start a game using a DOOM, DOOM2, TNT or PLUTONIA IWAD
2. Brightmaps will not be working.

**Comment by coelckers** [comment]
Works fine for me. Are you sure you haven't switched them off in the launcher window?

**Comment by blackhand1001** [comment]
> Works fine for me. Are you sure you haven't switched them off in the launcher window?

Yes the button is checked and the brightmaps.pk3 is loaded. The issue is the game does not make use of these bright maps. I suspect something is messed up with the iwad filtering for applying the gldefs etc.

**Comment by blackhand1001** [comment]
I suspect it is this commit that broke it. 

https://github.com/ZDoom/gzdoom/commit/21acb9d467b8f3406ab5031c8bff8510dc16f591

**Comment by Blue-Shadow** [comment]
Just like Graf, it works for me, too. (Screenshot taken using 4.12.2)


![Screenshot_Doom_20241014_105944](https://github.com/user-attachments/assets/962bdbba-f331-4193-8030-b9d9bd1f8c37)

**Comment by blackhand1001** [comment]
That green glow doesn't use gldef brightmaps. It works in my end as well. Check for the barrel glow and the red glowing eyes. They are not working.

**Comment by blackhand1001** [comment]
Yup that's that was it broken. I edited that pk3 to use the new lump filter names and it works now. The old names are broken in the current builds so old mods that use the old filters will be broken.

**Comment by NightFright2k19** [comment]
Already using the new lump filter system since years:
https://github.com/NightFright2k19/gzdoom_brightmaps/tree/master/bmplus_spritefix/filter
